ext/GeometryOpsDimensionalDataExt/GeometryOpsDimensionalDataExt.jl

```diff
@@ -0,0 +1,21 @@
+module GeometryOpsDimensionalDataExt
+
+import DimensionalData as DD
+import GeometryOps as GO
+import GeoInterface as GI
+
+function GO.polygonize(A::DD.AbstractDimArray; dims=(DD.X(), DD.Y()), crs=GI.crs(A), kw...)
```

Comment:
What happens with the dims that aren't given (time etc)?

Reply:
Yeah I haven't done that yet, this should be WIP.

But I image we make a feature collection with a feature for each slice, and put the lookup values as properties?

Reply:
Actually this is a case for R style vector data cubes.

But we can just broadcast over the other dimensions and get a DimArray of X/Y geometries. 

We should be able to write that directly as a FeatureCollection with GeoJSON/ArchGDAL/Shapefile without any more work as long as we name the array `:geometry`.

Reply:
Do we then want to add an assert here that the dims can only have length 2?